docs/connector-development/testing-connectors/connector-acceptance-tests-reference.md

```diff
@@ -362,3 +362,36 @@ acceptance_tests:
 
 We cache discovered catalogs by default for performance and reuse the same discovered catalog through all tests.
 You can disable this behavior by setting `cached_discovered_catalog: False` at the root of the configuration.
+
+## Additional Checks
+
+While not nesciacily related to SAT testing, Airbyte employs a number of additional checks which run on connector Pull Requests which check the following items:
```

Comment:
Nit: Mentioning SAT, should either be “connector acceptance testing” or maybe just omitting the first half of the sentence